org_users.go

```diff
@@ -0,0 +1,95 @@
+package gapi
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+)
+
+type OrgUser struct {
+	OrgId   int64  `json:"orgId"`
+	UserId  int64  `json:"userId"`
+	Email   string `json:"email"`
+	Login   string `json:"login"`
+	Role    string `json:"role"`
+}
+
+func (c *Client) OrgUsers(id int64) ([]OrgUser, error) {
```

Comment:
Could `id` be also called `orgId` as in other methods?

Reply:
Yes definitely